web/src/routes/Layout/index.tsx

```diff
@@ -319,10 +319,10 @@ class LayoutComponent extends React.Component<LayoutProps, LayoutState> {
                     <PublicRoute loading={userLoading} isLogged={isLoggedIn} path="/forgot_password" component={ForgotPasswordScreen} />
                     <PublicRoute loading={userLoading} isLogged={isLoggedIn} path="/accounts/password_reset" component={ChangeForgottenPasswordScreen} />
                     <PublicRoute loading={userLoading} isLogged={isLoggedIn} path="/email-verification" component={EmailVerificationScreen} />
-                    <Route path="/404" component={RestrictedScreen} />
-                    <Route path="/500" component={MaintenanceScreen} />
+                    <Route path="/restriction" component={RestrictedScreen} />
```

Comment:
i guess it's okay to keep /404 route

Reply:
we can't because we create func with this name on sonic side